pelican/log.py

```diff
@@ -203,7 +223,10 @@ def get_formatter():
         return TextFormatter()
 
 
-def init(level=None, handler=logging.StreamHandler(), name=None):
+def init(level=None, warnings_fatal=False, errors_fatal=False,
+         handler=logging.StreamHandler(), name=None):
+    FatalLogger.warnings_fatal = warnings_fatal
+    FatalLogger.errors_fatal = errors_fatal or warnings_fatal
```

Comment:
Isn't this backwards? `warnings_fatal` should exit if warnings or errors occur, not the other way around.